{"name": "test_failed_login", "status": "passed", "start": 1741270651044, "stop": 1741270651342, "uuid": "28af2d9b-264d-47ca-a735-04cd07425911", "historyId": "06b4296db8b878c7965c434c2acf5e7d", "testCaseId": "06b4296db8b878c7965c434c2acf5e7d", "fullName": "tests.api.test_auth#test_failed_login", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "DESKTOP-GIDTMH7"}, {"name": "thread", "value": "3360-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_auth"}]}